views/emails/email_verification.hbs

```diff
@@ -1,16 +1,35 @@
- <table width="100%" border="0" background="http://www.sputnik9.nl/img/bg4.jpg"
-style="background-size: cover; font-family: Arial, sans-serif;padding: 0 10% 30px;text-align: center;">
+<table width="100%" border="0" background="http://www.sputnik9.nl/img/bginvert.jpg"
+style="background-size: cover; font-family: Helvetica, Arial, sans-serif;text-align: center;">
+  
+  <tr style="width:100%; height:60px; background-color: #fde634;">
+    <td>
+      <h1 style="color: #111111; margin-top: 20px;display: inline-block;">Hoi, {{name}}!</h1>
+    </td>
+  </tr>
+
   <tr>
     <td>
+      <h3 style="color: #111111">Welkom bij Sputnik9!</h3>
+      <p style="color: #1111111">Bedankt voor het aanmaken van jouw Sputnik 9 account!</p>
+      <p style="color: #111111">Om jouw account te kunnen activeren, hebben we eerst een bevestiging van jouw e-mail adres nodig.</p>
+    </td>
+  </tr>
 
-      <h1 style="color: #fafafa;margin-top: 20px;display: inline-block;">Hoi, {{name}}!</h1>
-      <h3 style="color: #fafafa;">Welkom bij Sputnik9!</h3>
-      <p style="color: #fafafa;">Bedankt voor het aanmaken van jouw Sputnik 9 account!</p>
-      <p style="color: #fafafa;">Om jouw account te kunnen activeren, hebben we eerst een bevestiging van jouw e-mail adres nodig.</p>
+  <tr>
+    <td>
+      <img src="https://www.sputnik9.nl/img/sputniklogo.svg" width="100px"/>
```

Comment:
Gmail is geen teamplayer op vlag van svgs. Ik heb er al een png van gemaakt, mag even aangepast worden.